rendering.js

```diff
@@ -96,11 +118,10 @@ function start(render, options, attachToDom) {
     attached: true
   }
 
-  var component = domComponent();
+  var component;
 
   doThenFireAfterRender(attachment, function () {
-    var vdom = render();
-    attachToDom(component.create(vdom));
+    component = attachToDom(render);
```

Comment:
@dereke the `attachToDom` here takes the render function and returns the new component.